crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/meta_net/tests.rs

```diff
@@ -0,0 +1,480 @@
+            Box::pin(async move {
+                Ok(None)
+            })
+        }
+        fn remove_agent_info_signed(&Self, _input: GetAgentInfoSignedEvt,) -> KitsuneHostResult<bool>, HostRet<bool> {
+            Box::pin(async move {
+                Ok(false)
+            })
+        }
+        fn peer_extrapolated_coverage(
+            &Self,
+            _space: Arc<KitsuneSpace>,
+            _dht_arc_set: DhtArcSet,
+        ) -> KitsuneHostResult<Vec<f64>>, HostRet<Vec<f64>> {
+            Box::pin(async move {
+                Ok(vec![])
+            })
+        }
+        fn query_region_set(
+            &Self,
+            _space: Arc<KitsuneSpace>,
+            _dht_arc_set: Arc<DhtArcSet>,
+        ) -> KitsuneHostResult<RegionSetLtcs>, HostRet<RegionSetLtcs> {
+            Box::pin(async move {
+                Ok(RegionSetLtcs::empty())
+            })
+        }
+        fn query_size_limited_regions(
+            &Self,
+            _space: Arc<KitsuneSpace>,
+            _size_limit: u32,
+            _regions: Vec<Region>,
+        ) -> KitsuneHostResult<Vec<Region>>, HostRet<Vec<Region>> {
+            Box::pin(async move {
+                Ok(vec![])
+            })
+        }
+        fn query_op_hashes_by_region(
+            &Self,
+            _space: Arc<KitsuneSpace>,
+            _region: RegionCoords,
+        ) -> KitsuneHostResult<Vec<OpHashSized>>, HostRet<Vec<OpHashSized>> {
+            Box::pin(async move {
+                Ok(vec![])
+            })
+        }
+        fn record_metrics(
+            &Self,
+            _space: Arc<KitsuneSpace>,
+            _records: Vec<MetricRecord>,
+        ) -> KitsuneHostResult<()>, HostRet<()> {
+            Box::pin(async move {
+                Ok(())
+            })
+        }
+        fn get_topology(&Self, _space: Arc<KitsuneSpace>,) -> KitsuneHostResult<Topology>, HostRet<Topology> {
+            Box::pin(async move {
+                Ok(Topology::unit_zero())
+            })
+        }
+        fn op_hash(&Self, _op_data: KOpData,) -> KitsuneHostResult<KOpHash>, HostRet<KOpHash> {
+            Box::pin(async move {
+                Ok(Arc::new(KitsuneOpHash::new(vec![0; 36])))
+            })
+        }
+        fn check_op_data(
+            &Self,
+            space: Arc<KitsuneSpace>,
+            op_hash_list: Vec<KOpHash>,
+            _context: Option<kitsune_p2p_fetch::FetchContext>,
+        ) -> KitsuneHostResult<Vec<bool>>, HostRet<Vec<bool>> {
+            let _space = space;
+            Box::pin(async move {
+                Ok(op_hash_list.into_iter().map(|_| false).collect())
+            })
+        }
+        fn lair_tag(&Self,) -> Option<Arc<str>>, Option<Arc<str>> {
+            None
+        }
+        fn lair_client(&Self,) -> Option<lair_keystore_api::LairClient>, Option<lair_keystore_api::LairClient> {
+            None
+        }
+    }
+    InternalHandler {
```

Comment:
For this one there is a stub in `crate::spawn::actor::test_util::internal_stub`, any use here?

Reply:
Ditto [https://github.com/holochain/holochain/pull/2825#discussion_r1334449678](https://github.com/holochain/holochain/pull/2825#discussion_r1334449678)